### DISCUSSION TITLE
Improve Article to Admin Profile Flow

### DISCUSSION THREAD

**Discussion by FounderCasey** [forem/forem#16906] [Feature Requests]
> Please note that this request is for the admin experience.

**Issue**:
Currently, admins of all Forems have two ways to get to a users admin profile, `/admin/users/***`, they either navigate to the users profile page and append `/moderate` to the URL, which navigates them properly, or they have to go to the admin overview and search for the user via username. These two options work but can be tedious.

**Feature**:
Inside the mod panel and under the `Admin actions` dropdown I suggest that we add a button titled "Admin Page" that quickly and easily takes the user the selected users admin page.

![Screen Shot 2022-03-17 at 7 06 43 PM](https://user-images.githubusercontent.com/19507316/158923755-a763a8f7-b1cb-453f-adc5-f067e53a9177.png)

**Comment by Ridhwana** [comment]
I really like this idea @FounderCasey. It sounds like it would be useful

What do you think about changing the following from the mock:
1. Change it from a button a link. This is for accessibility reasons which you can read more about in this [conversation](https://forem-team.slack.com/archives/CSY9DBB2B/p1635516445006400)).
2. Change the text of the link to "View members profile" or something similar.

cc @JennieOcken to ultimately approve the feature request.

**Comment by ludwiczakpawel** [comment]
Great idea!

I think we could take it a step further even and add "admin-oriented" links in few other places as well just to keep them always handy:

![image](https://user-images.githubusercontent.com/108287/160077150-14deebf4-6a7f-4eef-adf7-34ec1e7ebf9c.png)
